src/features/tags/components/TagItem.js

```diff
@@ -1,23 +1,21 @@
 import React, { Component } from "react";
-import { NavLink } from "react-router-dom";
 import { connect } from "react-redux";
 import { getTagById } from "../selectors";
+import { Link } from "@mui/material";
 
 class TagItem extends Component {
   render() {
     const { tag, key } = this.props;
 
     return (
       <li className="menu__list-item" name={tag.id} key={key}>
-        <NavLink
-          exact
-          to={`/tag/${tag.id}`}
-          activeClassName="menu__link menu__link--active active"
-          className="menu__link"
+        <Link 
```

Comment:
The TagItem link needs to be controlled by React Router's Navlink so that there's not a whole page refresh when clicking this link. [Here's](https://mui.com/components/links/#third-party-routing-library) more info on how to accomplish this with MUI's Link component.

Also, this link display needs to look similar to the previous version (grey text and no underline). Right now it is blue with an underline (see below)
<img width="243" alt="Screen Shot 2021-10-29 at 3 58 34 PM" src="https://user-images.githubusercontent.com/37122448/139500897-a098ab4b-ff00-4eca-b466-f2d341750b51.png">